frontend/components/SingleDocument.js

```diff
@@ -0,0 +1,51 @@
+import React, {useEffect, useState} from "react";
+import * as PropTypes from "prop-types";
+import STYLES from "./SingleDocument.module.scss";
+
+const SingleDocument = ({id}) => {
+
+    const [docData, setDocData] = useState({});
+    const [loading, setLoading] = useState(true);
+    const [showText, setShowText] = useState(false);
+
+    useEffect(() => {
+        fetch(`/api/document/${id}`)
+            .then(response => response.json())
+            .then(data => {
+                setDocData(data);
+                setLoading(false);
+                console.log(data);
```

Comment:
If you feel like the rest of this is in a good set, let's get rid of any debugging logs.